insightfinder/assets/dashboards/ifdashboard.json

```diff
@@ -0,0 +1,292 @@
+{
+  "title": "InsightFinder Dashboard",
+  "description": "This dashboard provides a view of predicted or detected incidents from InsightFinder. \n\nAbout InsightFinder - \nInsightFinder Unified Intelligence Engine (UIE) platform provides human centered AI solutions for identifying incident root causes, an for predicting and preventing production incidents. Visit us at www.insightfinder.com to learn more.",
+  "widgets": [
+    {
+      "id": 5694616657083584,
+      "definition": {
+        "title": "InsightFinder",
+        "type": "group",
+        "background_color": "vivid_orange",
+        "show_title": true,
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 5461395352924928,
+            "definition": {
+              "type": "image",
+              "url": "https://uc199676fafed31e1fccaefa4f33.previews.dropboxusercontent.com/p/thumb/ABkKXnSMbkYVyHf_IvQh7X8OrdbS1MNwyYzWKtsUDUInabtlTmLM1cHcXOioSBF88vt9FhGtIi3pS8v2gRiz3xDNePvCl4r8ihkB3l1LN-qIzAmArOpbkMjXAbwqUrkWt2yfqo3-UUleKS7MulNdGtywzQ4MJfteo_HOTdiUStCmjQqRDRjhC5CU_9xz2npI8_7Cvb1dW2UJg_BsdgNGZ4XWH6RuJRGAM6TD38SeU76k5kxi1FESCJHQ-_sOwif2GLcTsyOttqtAkQYp4AQm6XEq_C1DlFTxXEMvEQWztqhYCeAvb8Cr-YigLxXsyRqKZQXoF5AOg5qBvTJaPxqSQcifnjVPUhDgbskTqibz-KsWD3YqA7iqfuPfJSPeWIGLgns/p.png?size=512x512&size_mode=1",
+              "sizing": "contain",
+              "has_background": false,
+              "has_border": true,
+              "vertical_align": "center",
+              "horizontal_align": "center"
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 2,
+              "height": 2
+            }
+          },
+          {
+            "id": 1510135480041762,
+            "definition": {
+              "type": "note",
+              "content": "This dashboard provided the view of predicted or detected incidents from InsightFinder.\n\nAbout InsightFinder - InsightFinder Unified Intelligence Engine (UIE) platform provides human centered AI solutions for identifying incident root causes, predicting and preventing production incidents. Visit us at www.insightfinder.com to learn more.\n\nhttps://insightfinder.com/datadog-integration/",
```

Comment:
```suggestion
              "content": "This dashboard provides a view of predicted or detected incidents from InsightFinder.\n\nAbout InsightFinder - InsightFinder Unified Intelligence Engine (UIE) platform provides human centered AI solutions for identifying incident root causes, and for predicting and preventing production incidents. Visit us at www.insightfinder.com to learn more.\n\nhttps://insightfinder.com/datadog-integration/",
```

Reply:
@ajainIF I just wanted to bring your attention to this comment as it's still unresolved

Reply:
@anawishnoff - you mean its not updated in the repository? I see it updated here - https://github.com/ajainIF/integrations-extras/blob/insightfinder-dd-integration/insightfinder/assets/dashboards/ifdashboard.json

Reply:
done